Source/WebCore/platform/graphics/cocoa/VideoMediaSampleRenderer.mm

```diff
@@ -61,6 +61,12 @@ - (void)resetUpcomingSampleBufferPresentationTimeExpectations;
     static inline CFTypeID typeID(void) { return PAL::CMSampleBufferGetTypeID(); }
 };
 
+#if ENABLE(LINEAR_MEDIA_PLAYER)
+@interface AVSampleBufferVideoRenderer (Staging_127455709)
+- (void)removeAllVideoTargets;
+@end
+#endif
```

Comment:
This staging should no longer be necessary since the SPI has been available in SDKs for a while at this point; instead we should just declare this in AVFoundationSPI.h for the sake of Public SDK builds.

Reply:
I'll be taking this in a follow up as here I'm really just moving code, and this should be going to current release